discovery/consul/consul_test.go

```diff
@@ -90,6 +90,7 @@ const (
 "NodeMeta": {"rack_name": "2304"},
 "ServiceID": "test",
 "ServiceName": "test",
+"ServiceMeta": {"version":"1.0.0"},
```

Comment:
Optional: For better coverage it might be worth adding two values here (this way you validate that there is really a loop)

Reply:
Done.